client/src/components/Form.js

```diff
@@ -32,12 +32,12 @@ const ButtonIcon = styled.button`
   }
 `;
 
-export const FormInput = ({ title, icon, alt }) => {
+export const FormInput = ({ title, icon, alt, onClick, onChange }) => {
```

Comment:
I would expect that `onClick` will be triggered if I click somewhere on the input.
Instead, it's fired on image click.

I would specify the naming:

```suggestion
export const FormInput = ({ title, icon, alt, onImageClick, onChange }) => {
```

Reply:
good point!